main.py

```diff
@@ -49,6 +74,10 @@ def draw():
 
     for entity in entities:
         WINDOW.blit(entity.sprite.img, entity.pos, entity.sprite.rect)
+        pygame.font.init()
+        myfont = pygame.font.SysFont('Comic Sans MS', 30)
```

Comment:
we should do these in init(), not every frame

Reply:
Done